src/dev/python/NativeCall.py

```diff
@@ -3,11 +3,11 @@
 from ctypes import *
 import os
 # for windows
-#dll_core = CDLL(os.getcwd() + '\dsn.core.dll')
-#dll_helper = CDLL(os.getcwd() + '\dsn.dev.python_helper.dll')
+dll_core = CDLL(os.getcwd() + '\dsn.core.dll')
```

Comment:
is it possible for us to have something like # ifdef _Win32 # elif defined (__linux__) #endif etc. to make sure it runs on multiple platforms?